assets/js/base/components/chip/test/__snapshots__/index.js.snap

```diff
@@ -55,67 +70,104 @@ exports[`Chip without custom wrapper should render custom aria label 1`] = `
   >
     Test
   </span>
-  <button
-    aria-label="Aria test"
-    className="wc-block-components-chip__remove"
-    disabled={false}
-    onClick={[Function]}
-    onKeyDown={[Function]}
-  >
-    🗙
-  </button>
 </li>
 `;
 
-exports[`Chip without custom wrapper should render defined radius 1`] = `
-<li
-  className="wc-block-components-chip wc-block-components-chip--radius-large"
+exports[`Chip with custom wrapper should render a chip made up of a div instead of a li 1`] = `
+<div
+  className="wc-block-components-chip wc-block-components-chip--radius-small"
 >
   <span
     aria-hidden={false}
     className="wc-block-components-chip__text"
   >
     Test
   </span>
+</div>
+`;
+
+exports[`RemovableChip should render custom aria label 1`] = `
+<li
+  className="is-removable wc-block-components-chip wc-block-components-chip--radius-small"
+>
+  <span
+    aria-hidden={false}
+    className="wc-block-components-chip__text"
+  >
+    <h1>
+      Test
+    </h1>
+  </span>
   <button
-    aria-label="Remove"
+    aria-label="Aria test"
     className="wc-block-components-chip__remove"
     disabled={false}
     onClick={[Function]}
     onKeyDown={[Function]}
   >
-    🗙
+    <svg
+      aria-hidden="true"
+      className="dashicon dashicons-arrow-down-alt2"
```

Comment:
🤔 hmm, this is a bit surprising, shouldn't this be the `no-alt` icon in the snapshot? Actually in looking closer, it looks like we have bad classnames in the `no-alt.js` file in our icon library folder. It could be handled in a separate issue (See #2846).

Reply:
Ah, good catch! I missed that. Will open a PR today fixing it.